tests/test_optical_model.py

```diff
@@ -0,0 +1,21 @@
+from catkit2.catkit_bindings import DataStream
+from catkit2.simulator import SimpleOpticalModel
+
+import hcipy
+import numpy as np
+
+
+def test_optical_model():
+    model = SimpleOpticalModel()
+    pupil_grid = model.pupil_grid
+
+    wf_in = hcipy.Wavefront(pupil_grid.ones())
+    model.set_wavefronts('pre_pupil', wf_in)
+
+    for i in range(5):
+        model.atmosphere.t += 0.01
+        model.purge_plane('pre_coro')
+
+        wf_out = model.get_wavefronts('science_camera')[0]
+
+        assert np.sum(wf_out.intensity) > 0
```

Comment:
I am dubious about this test - why doing 5 iterations if it's to test the same thing every time? Why not test that the `wf_out` has changed? ^^
Also I understand it's a low-key test, so I don't think that's very important.

Reply:
You're totally right, but the only thing to test here is that it runs at all. The assert is rather random 🤷